plugins/configuration.rb

```diff
@@ -175,6 +175,7 @@ def render(context)
             platform: platform,
             converter: converter
           )}
+          <p class="configuration-link" title="Click here to find your configuration file"><a href="/docs/configuration/" target="_blank"><i class="icon-question" />Find your configuration</a>
```

Comment:
```suggestion
          <p class="configuration-link" title="Click here to find your configuration file"><a href="/docs/configuration/" target="_blank"><i class="icon-question" /> find your configuration</a>
```

Reply:
Why lowercase it?

Reply:
I felt it looked weird, but still playing with it. Want to explore adding it to the top.

Reply:
I added it to the top first, did not look good IMO